README.md

```diff
@@ -24,32 +24,53 @@ steps:
 ```
 
 ## Using the plugin
-Build the container:
+
+Get a maintained build from
+[quay.io/arcalot/arcaflow-plugin-pcp](https://quay.io/repository/arcalot/arcaflow-plugin-pcp),
+or build the container locally:
 ```
-docker build . -t arcaflow-plugin-pcp
+podman build . -t arcaflow-plugin-pcp
 ```
 
 Run with the provided example input:
 ```
-cat configs/pcp_example.yaml | docker run -i --rm arcaflow-plugin-pcp -f -
+podman run -i --rm arcaflow-plugin-pcp -s start-pcp -f - < configs/pcp_example.yaml
 ```
 
-***Note:** This plugin is designed to be used as a container image built with
-the provided Dockerfile. Using the python directly on a target system will
-likely prove problematic*
+> [!NOTE]
+> This plugin is designed to be used as a container image built with the provided
+> Dockerfile. Using the python directly on a target system will likely prove problematic
+
+### Post-Processing Only
+
+The `post-process` plugin step can be run independently to convert an existing PCP
+archive file into the same structured format that the main `start-pcp` plugin step
+provides. You will need to provde the path to the existing archive file as the
+`archive_path` input to the `post-process` step. When the plugin is run in a container,
+you will need to bind-mount the archive file path to the container so that the path is
+available for processing, and the `archive_path` will be in the context of the container
+namespace. For example:
+
+```
+podman run -i --rm -v /example/local_pcp_archive/:/pcp_archive/:Z \
+arcaflow-plugin-pcp -s post-process -f - << EOF
+archive_path: /pcp_archive/pmlogger-out
+EOF
+```
```

Comment:
I would be inclined to make the explanation more concrete.  For instance,
> In order to make the archive file available to the plugin when it is run in a container, you will need to bind-mount the archive file path to the container and specify the `archive_path` in the context of the container namespace. For example, to post-process an archive in `/example/local_pcp_archive/pmlogger-out` use the following command:
> ```
> podman run -i --rm -v /example/local_pcp_archive/:/pcp_archive/:Z \
> arcaflow-plugin-pcp -s post-process -f - << EOF
> archive_path: /pcp_archive/pmlogger-out
> EOF
> ```

And, given that there is only one line of input, consider using a "here string" instead of a "here document":
```
podman run -i --rm -v /example/local_pcp_archive/:/pcp_archive/:Z \
arcaflow-plugin-pcp -s post-process -f - <<< "archive_path: /pcp_archive/pmlogger-out"
```